src/main/java/duke/tasklist/task_types/Event.java

```diff
@@ -0,0 +1,31 @@
+package duke.tasklist.task_types;
+
+/**
+ * Represents a <code>task</code> object that contains a string holding the <code>event</code> time
+ * from a specific length stated by the user.
+ * 
```

Comment:
Consider removing the blank line at line 6 to keep a consistent format for the header Javadoc of each class.